holmes_operator/handlers/healthcheck.py

```
@@ -0,0 +1,254 @@
+    set_healthcheck_running,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@kopf.on.create("holmesgpt.dev", "v1alpha1", "healthchecks")
+async def on_healthcheck_create(
+    spec: Dict[str, Any],
+    name: str,
+    namespace: str,
+    uid: str,
```

Ruff (0.14.14)

[warning] Unused function argument: `uid`

(ARG001)